libs/providers/flagd-web/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@openfeature/flagd-web-provider",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "scripts": {
+    "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [ \"$(npm show $npm_package_name@$npm_package_version version)\" = \"$(npm run current-version -s)\" ]; then echo 'already published, skipping'; else npm publish --access public; fi",
+    "current-version": "echo $npm_package_version"
+  }
```

Comment:
@james-milligan do we need any runtime dependencies here?

Reply:
i dont believe so

Reply:
I would think `node-cache` at least?

Reply:
I think we need:

`@bufbuild/connect-web`
`@bufbuild/protobuf`
`ts-md5`
and `node-cache`

Reply:
@james-milligan @beeme1mr 

I've added `buildableProjectDepsInPackageJsonType` to flagd-web's project.json, as well as the templates/generators. This parses the TS/JS and automatically includes the detected dependencies. It added:

```
  "dependencies": {
    "@bufbuild/connect-web": "^0.2.1",
    "ts-md5": "^1.3.1",
    "node-cache": "^5.1.2"
  }
```

To the flagd-web package.json.

@beeme1mr I also added a peerDep for `"@openfeature/js-sdk"` to the templates/generators.